configurator/configurator-impl/src/main/java/org/codice/ddf/admin/configurator/impl/ConfigValidator.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.configurator.impl;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang.StringUtils;
+
+import com.google.common.io.Files;
+
+class ConfigValidator {
+    static void validateString(String bundleSymName, String msg) {
+        if (StringUtils.isBlank(bundleSymName)) {
+            throw new IllegalArgumentException(msg);
```

Comment:
I would much rather return an object indicating wether the input was valid or not instead of throwing an exception. I think long term we should also collapse all the validation into a service so it can be used across a system.  I'm concerned about adding tech debt/forgetting to change this if we do go that route, so it might be worth leaving a todo or creating a ticket to follow up on this.